### DISCUSSION TITLE
ReplyKeyboardMarkup not showing up after 1st appearance

### DISCUSSION THREAD

**Discussion by atlterry** [aiogram/aiogram#1379] [Q&A] [answered]
python: 3.10.2
aiofiles==23.2.1
aiogram==3.2.0
aiohttp==3.9.1

I upgraded aiogram to 3.2 from 2.x and update my code for the ReplyKeyboardMarkup
`        await message.reply(f"Your message is: {message.text}"
                            , disable_web_page_preview = True
                            , reply_markup=ReplyKeyboardMarkup(keyboard=keyboard, resize_keyboard=True))`

![image](https://github.com/aiogram/aiogram/assets/19489158/fe7ef4e9-03d3-4cc3-99b1-fade8e4dbc7d)
(1) Is there any way to suppress the telegram messages?
(2) the menu icon shows up fine
![image](https://github.com/aiogram/aiogram/assets/19489158/d708f957-2a6a-41f2-81d6-4ae2b04e23e8)

After selecting an option ...
(3) The menu is no longer available
(4) a default / telegram message appears !! Is there a way to suppress this message?
![image](https://github.com/aiogram/aiogram/assets/19489158/bcc079d0-30de-44c0-9aa6-b935920af346)

No errors reported on the console!

Thanks in advance

**Comment by JrooTJunior** [comment]
All in your hands. You didn't provide any code to help you with.

**Reply by atlterry** [reply]
Sorry - here you go...
[Pastebin code](https://pastebin.com/embed_js/nXeLyzEw)
Topic-1,2,3.html etc are simple html content files

**Comment by atlterry** [comment, marked as answer]
I deleted & added the bot and the message is not showing anymore !
